pcsx2-qt/SettingWidgetBinder.h

```diff
@@ -1076,17 +1076,22 @@ namespace SettingWidgetBinder
 	}
 
 	static inline void BindSliderToIntSetting(SettingsInterface* sif, QSlider* slider, QLabel* label, const QString& label_suffix,
-		std::string section, std::string key, s32 default_value)
+		std::string section, std::string key, s32 default_value, bool clamp_value = false)
 	{
-		const s32 global_value = Host::GetBaseIntSettingValue(section.c_str(), key.c_str(), default_value);
+		s32 global_value = Host::GetBaseIntSettingValue(section.c_str(), key.c_str(), default_value);
+
+		//if enabled, clamp in case setting was updated manually using INI
```

Comment:
We might want to do this unconditionally anyway.. can't think of many reasons why you wouldn't want to clamp to the UI range.

Reply:
Agreed, just didn't want to step on anyone's toes in case they did. I'll make the change to remove the condition and can add it back if needed :)